foxy/ci-nightly-extra-rmw-release.yaml

```diff
@@ -28,6 +28,7 @@ jenkins_job_label: ci-agent
 jenkins_job_priority: 47
 jenkins_job_schedule: 15 23 * * *
 jenkins_job_timeout: 300
+package_selection_args: '--packages-ignore'
```

Comment:
Is it okay that this has no trailing packages? Maybe just remove the field.

Reply:
Good catch: removed in 51c569326c6d2e55c96f029579a99fd24ec89f0a.